js/main.js

```diff
@@ -20,4 +20,94 @@ $form.addEventListener('submit', function (event) {
   data.entries.unshift(newEntry);
   $entryImage.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+  $list.prepend(renderEntry(newEntry));
+  viewSwap('entries');
+  if (data.entries !== null) {
+    toggleNoEntries();
+  } else {
+    toggleNoEntries();
+  }
+});
+
+function renderEntry(entry) {
+
+  const $row = document.createElement('li');
+  const $columnPhoto = document.createElement('div');
+  const $columnNotes = document.createElement('div');
+  const $image = document.createElement('img');
+  const $header = document.createElement('h2');
+  const $notes = document.createElement('div');
+
+  $row.className = 'row';
+  $columnPhoto.className = 'column-full column-half';
+  $image.className = 'photo-url';
+  $image.setAttribute('alt', 'entry image');
+  $columnNotes.className = 'column-full column-half';
+  $header.className = 'title';
+  $notes.className = 'notes';
+
+  $row.appendChild($columnPhoto);
+  $columnPhoto.appendChild($image);
+  $row.appendChild($columnNotes);
+  $columnNotes.appendChild($header);
+  $columnNotes.appendChild($notes);
+
+  $image.setAttribute('src', entry.photoURL);
+  $header.textContent = entry.title;
+  $notes.textContent = entry.notes;
+
+  return $row;
+}
+
+const $list = document.querySelector('ul');
+
+document.addEventListener('DOMContentLoaded', function () {
+  for (let i = 0; i < data.entries.length; i++) {
+    $list.appendChild(renderEntry(data.entries[i]));
+  }
+  const lastView = localStorage.getItem('currentView');
+  if (lastView) {
+    viewSwap(lastView);
+  }
+  if (data.entries !== null) {
+    toggleNoEntries();
+  } else {
+    toggleNoEntries();
+  }
+});
+
+function toggleNoEntries() {
+  const $noEntry = document.querySelector('.no-entries');
+  $noEntry.className = 'no-entries-hidden';
+}
```

Comment:
Eventually we will want to show this, so we should be toggling this, rather than just always setting to `hidden`. Eventually you will be able to delete an entry, so if we delete all of entries, we need to make sure this text comes back.

We should be conditionally checking if `data.entries` has any items in or not, and based off of that, we should either give a class of `hidden` or remove it.